index.html

```diff
@@ -67,6 +121,57 @@
     // Submit the form
     $("#dlForm").submit();
   })
+
+  $("#plus").click(newDoc);
+  $("#minus").click(function() {
+    // TODO: replace this with something prettier
+    if (!confirm('OK to delete?')) {
+      return;
+    }
+    var n = $('.cur').next();
+    if (n.length === 0) {
+      n = $('.cur').prev();
+    }
+    $('.cur').remove();
+    delete localStorage[cur];
+    if (n.length>0) {
+      cur = n[0].id;
+      load(cur);
+    } else {
+      newDoc();
+    }
+  });
+
+  // Automatically run the transform after a keypress, throttled with a timer.
+  $("#in").keyup(function() {
+    if (state.timer) {
+      clearTimeout(state.timer);
+    }
+    state.timer = setTimeout(function(){
+      state.timer = null;
+      save();
```

Comment:
Is this actually what you want?  I was expecting "throttle" to mean "don't save more than once per X seconds", not "save every X seconds".  So something like the following:

```
var now = new Date();
if (now - lastSaved < SAVE_TIMEOUT) { save(); lastSaved = now; }
```

That saves you a lot of setTimeout / clearTimeout thrash.